src/Microsoft.AspNet.Hosting.Abstractions/EnvironmentName.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Hosting
+{
+    public static class EnvironmentName
```

Comment:
Yeah I don't think an enum is the best option here for the reason @hishamco mentions. These really are fundamentally strings.